src/Console/Command.php

```diff
@@ -160,7 +167,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $fileReportList = new FileReportList();
         $printer = new Printer\Console();
+        $whitelist = $this->getFileOption($input->getOption('whitelist'));
+
         foreach ($finder as $file) {
+            if (count($whitelist) > 0 && !in_array($file->getRelativePathname(), $whitelist)) {
```

Comment:
I did look there but it seems to get messy. When this option is enabled only the files in the whitelist which would normally be scanned should be shown.

Reply:
The thing is that $finder must have all files it will search without any further conditions :) I think PHPFinder has methods you need to exclude these files.